swan-lake/learn-the-platform/test-document-the-code/test-ballerina-code/execute-tests.md

```diff
@@ -162,6 +162,23 @@ Also, dump the test results in the JSON format.
 $ bal test --test-report --code-coverage --coverage-format=xml
 ```
 
+Exclude files or modules from the code coverage report. The exclusion list can be provided as a comma-separated list of absolute or relative paths. Glob patterns such as `*` and `**` can be used in the paths.
```

Comment:
```suggestion
Now, you can exclude files or modules from the code coverage report. The exclusion list can be provided as a comma-separated list of absolute or relative paths. Glob patterns such as `*` and `**` can be used in the paths.
```

Reply:
We followed the same convention(List, Exclude, Run and so on) for other flags too.